.travis.yml

```diff
@@ -11,7 +11,8 @@ env:
     - SCAN_WORKSPACE="SwiftSampleProject/SwiftSampleProject.xcworkspace"                 SCAN_SCHEME="SwiftyVokoder"              SCAN_SDK="iphonesimulator"
     - SCAN_WORKSPACE="Paging Data Source Example/Paging Data Source Example.xcworkspace" SCAN_SCHEME="Paging Data Source Example" SCAN_SDK="iphonesimulator"
 script: bundle exec scan
-after_script:
+after_success:
+  - bundle exec pod repo update # Travis isn't finding the latest VOKUtilities release: must have an old repo cached
```

Comment:
We weren't. I'm just saying, it can't be THAT bad, because we run `pod install` locally all the time and I don't remember too many gripes about how long that was taking. This command took 9 seconds on my machine just now, so we'll see what the impact is on Travis.